lib/datadog/ci/ext/environment.rb

```diff
@@ -85,29 +90,27 @@ def extract_appveyor(env)
             TAG_PIPELINE_URL => url,
             TAG_JOB_URL => url,
             Datadog::Ext::Git::TAG_BRANCH => branch,
-            Datadog::Ext::Git::TAG_TAG => tag
+            Datadog::Ext::Git::TAG_TAG => tag,
+            Datadog::Ext::Git::TAG_COMMIT_AUTHOR_NAME => env['APPVEYOR_REPO_COMMIT_AUTHOR'],
+            Datadog::Ext::Git::TAG_COMMIT_AUTHOR_EMAIL => env['APPVEYOR_REPO_COMMIT_AUTHOR_EMAIL'],
+            Datadog::Ext::Git::TAG_COMMIT_MESSAGE => env['APPVEYOR_REPO_COMMIT_MESSAGE_EXTENDED']
           }
         end
 
         def extract_azure_pipelines(env)
+          pipeline_url = job_url = nil
           if env['SYSTEM_TEAMFOUNDATIONSERVERURI'] && env['SYSTEM_TEAMPROJECTID'] && env['BUILD_BUILDID']
             base_url = "#{env['SYSTEM_TEAMFOUNDATIONSERVERURI']}#{env['SYSTEM_TEAMPROJECTID']}" \
               "/_build/results?buildId=#{env['BUILD_BUILDID']}"
+
```

Comment:
Not sure I understand, sorry.
Are you suggesting we use `fetch` instead of `[]` because fetch fails when the key is not present, and in this case we know it's present?

If that's the case, I don't see much value in this change, given `[]` is more succinct.

Reply:
(I refactored this method btw, I don't think the original comment applies, but I'd still like to know what your original feedback was.)